### PR TITLE
Use the latest Rubies on Travis CI

### PR DESCRIPTION
**Pull request by yhirano55**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,9 +7,9 @@ rvm:
   - jruby-head
   - 2.1.10
   - 2.2.9
```
Comment (pocke) [inline .travis.yml]: Probably we should update Ruby 2.2 to 2.2.10 also.
https://www.ruby-lang.org/en/news/2018/03/28/ruby-2-2-10-released/

Comment (yhirano55) [reply .travis.yml]: Oops, thank you. I'll fix it.

**Comment by bbatsov** [comment]
> I wish that TravisCI would fix the two digit form of Ruby version so that we could avoid having to make these updates.

Perhaps it's time to start evaluating some alternative to Travis. Often I hear that Circle CI is more robust and less painful to use.

**Comment by pocke** [comment]
Definitely. Probably we can solve the slow test in JRuby also. I guess it is entropy problem.
I'm not sure Circle CI solves this problem, but we can try it :laughing: 
Maybe related: https://github.com/travis-ci/travis-ci/issues/1494

**Pull request merged**